android/package.json

```diff
@@ -15,13 +15,13 @@
 	],
 	"minSDKVersion": "16",
 	"vendorDependencies": {
-		"android sdk": ">=23 <=25.x",
-		"android build tools": ">=23 <=25.x",
-		"android platform tools": ">=17 <=25.x",
+		"android sdk": ">=25 <=26.x",
+		"android build tools": ">=25 <=26.x",
+		"android platform tools": ">=25 <=26.x",
 		"android tools": "<=26.x",
-		"android ndk": ">=r8e <=r9",
+		"android ndk": ">=r11c <=r15c",
 		"node": ">=4.0 <=7.x",
-		"java": "<=1.8.x"
+		"java": ">=1.8.x"
```

Comment:
**Question:**
Our "node scons.js build" command currently compiles our core Java code using Java version 1.6, regardless of what's installed. It would be nice if we can bump it up to Java 1.7 so that we can support new Java features such as the try-with-resource blocks...
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Do you happen to know where in our CLI or build scripts this is controlled. We can make this a separate PR I suppose, but since we're now requiring a higher JDK version here, this might be a good opportunity to use newer Java features. :)